client/src/components/TeamList.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GhostbusterButton from './GhostbusterButton';
+import Team from './Team';
+
+const TeamList = (props) => {
+  const {
+    checkProjects,
+    loading,
+    projects,
+    showSegment,
+  } = props;
+  const style = loading
```

Comment:
Can you add some whitespace in here so it's easier to scan the code to see what's up?